Source/Core/Core/BootManager.cpp

```diff
@@ -319,6 +321,7 @@ bool BootCore(std::unique_ptr<BootParameters> boot, const WindowSystemInfo& wsi)
   }
 
   StartUp.m_GPUDeterminismMode = ParseGPUDeterminismMode(StartUp.m_strGPUDeterminismMode);
+  StartUp.m_freelook = Config::Get(Config::GFX_FREE_LOOK);
```

Comment:
Not sure if I missed this somewhere, but since this is set once on boot; do we lock the setting so it cannot be changed while a game is running?
Also, on the other hand: is there a reason why we have to (or rather: want to) prevent this being enabled while in-game?

Reply:
This shouldn't impede changing while in game (at least I didn't think?) this is just so that the controllers can be initialized/setup properly (just like with Wii/Gamecube)...though maybe this needs to be handled differently since you can turn it off.

Reply:
Still thinking of how best to resolve this.  Appreciate you bringing it up

Reply:
I mean enabling/disabling freelook (setting `GFX_FREE_LOOK`), which is then handled in `BootCore::EmuThread`. It is read there twice; once during the normal boot and once in the `ScopeGuard` to shut things down later. It _might_ be ok if you only set it here, since it should have the same value in both; but this also means enabling it while a game is running (and it was set to `false` previously) won't have it initialized.

Reply:
Ah I see.  I will have to see how Wiimote/Gamecube controllers get dynamically connected then.  I don't want users to remember to set this during startup (that's not how it currently works!)

Reply:
I've fixed this by making the freelook controllers _always_ initialized and instead have their individual state controlled by the camera control type.  A value of "None" will turn off that camera.